src/Artsy/Relay/createMockNetworkLayer.ts

```diff
@@ -1,12 +1,16 @@
+import { readFileSync } from "fs"
 import { IMocks } from "graphql-tools/dist/Interfaces"
 import getNetworkLayer from "relay-mock-network-layer"
 import { Network } from "relay-runtime"
-import schema from "../../../data/schema.json"
+
+// TODO: This doesn't work in client side rendering
+const schemaDSL = readFileSync("data/schema.graphql", "utf8")
```

Comment:
Ah yes, I bet it’s because you’re trying to read a file, which you can’t do from a browser. I now faintly remember this being a reason for me to choose the json file at the time.

I’m sure that webpack has solutions for embedding data.

Reply:
Maybe there’s a more efficient solution, but I think that the simple ‘replace’ plugin could do the job by just replacing this call with a string literal that contains the GraphQL source.

Reply:
It's not much hassle to do in webpack: https://github.com/webpack/docs/wiki/how-to-write-a-loader

Then we could just `require('<any-old-file>.graphql')`

Reply:
in fact several already exist  e.g. https://github.com/samsarahq/graphql-loader